spec/PhpSpec/Loader/Node/ExampleNodeSpec.php

```diff
@@ -22,7 +22,15 @@ function it_provides_a_link_to_title()
         $this->getTitle()->shouldReturn('example node');
     }
 
-    function it_provides_a_link_to_function($function)
+    function it_can_set_a_title()
+    {
+      $this->setTitle('node example');
+      $this->getTitle()->shouldReturn('node example');
+      $this->setTitle('example node');
```

Comment:
Why are you setting the title again here, after an assertion?

Reply:
This is indeed not necessary, do you want me to remove it ?